shiny/workflowPlot/server_files/model_data_plots_server.R

```diff
@@ -43,10 +43,10 @@ observeEvent(input$var_name_modeldata, {
 
 # Check that new units are parsible and can be used for conversion
 observeEvent(input$units_modeldata,{
-  if(udunits2::ud.is.parseable(input$units_modeldata)){
+  if(units::install_unit(input$units_modeldata)){
```

Comment:
As we discussed in PEcAn.DB, install_units isn't the right approach here: `ud.is.parseable` checks if the udunits library knows how to handle the unit, `install_unit` declares the unit as valid even if the udunits library has never seen it before. 

What we did in PEcAn.DB was to wrap `as_unit()` in a tryCatch, and that's probably the right approach here too. A shorter but probably much harder to understand hack would be `if( PEcAn.utils::ud_convert(1, input$units_modeldata, input$units_modeldata)) { #test if unit parseable by trying to convert to itself`

Reply:
latter is a very cool approach indeed! @nanu1605 please go ahead and try this. If you're confused, ping me or Chris.

Reply:
I really like @infotroph 's latter suggestion, it's a very cool approach but now that you've written a replacement function for `ud.is.parseable`, go ahead and replace these calls with the `unit_is_parseable` function once you get the green light from other reviewers.

Reply:
After thinking more, I think my latter suggestion is a bad one -- it's clever but too easy to misunderstand. I can easily see a future reader (including myself in a year!) saying "The input and output units are the same, so this line does nothing! I can delete it!", so defining a new function with a clear name seems preferable.